storage/innobase/row/row0merge.cc

```diff
@@ -4178,7 +4178,7 @@ row_merge_file_destroy_low(
 	const pfs_os_file_t& fd)	/*!< in: merge file descriptor */
 {
 	if (fd != OS_FILE_CLOSED) {
-		os_file_close(fd);
+		my_close(fd, MYF(0));
```

Comment:
`pfs_os_file_t` is supposed to wrap a native file handle, which on Windows is `HANDLE`, not `int fd`. I think that we should try to keep `row_merge_file_create_low()` to work on native file handles, not wrapped by `my_sys.h`. Alternatively, all I/O on these temporary file handles has to be rewritten.

Reply:
Added da48f97f17e to revert this chunk and continue to use mysys for allocating tmp file. Per https://github.com/MariaDB/server/pull/657#issuecomment-375739279 mysys is quite ripe for changing File to HANDLE so a duplication of mysys tempfile to innodb seems a little excessive.